tsne/bh_sne_src/tsne.h

```diff
@@ -53,7 +53,7 @@
 
```

Comment:
I think in windows we also need to define it as `dllimport` when included from places which are not this binary.  If we're only building this into a binary, not as a dll on windows, then we shouldn't be using `dllexport`.

Reply:
Given how we're using this, it wasn't strictly necessary. But I did it anyway.